shared/keystore/keystore.go

```diff
@@ -80,6 +81,38 @@ func (ks Store) GetKey(filename, password string) (*Key, error) {
 	return DecryptKey(keyjson, password)
 }
 
+// GetKeys from directory using the prefix to filter relevant files
+// and a decryption password.
+func (ks Store) GetKeys(directory, fileprefix, password string) ([]*Key, error) {
+	// Load the key from the keystore and decrypt its contents
+	// #nosec G304
+	files, err := ioutil.ReadDir(directory)
+	if err != nil {
+		return nil, err
+	}
+	keys := []*Key{}
+	i := 0
+	for _, f := range files {
+		n := f.Name()
+		filePath := filepath.Join(directory, n)
+		filePath = filepath.Clean(filePath)
+		if f.Mode().IsRegular() && strings.Contains(n, strings.TrimPrefix(fileprefix, "/")) {
+			keyjson, err := ioutil.ReadFile(filePath)
```

Comment:
```suggestion
                        // #nosec G304
			keyjson, err := ioutil.ReadFile(filePath)
```

Reading a file by variable name is a low security risk, so you need to add the exclusion comment.